src/test/java/com/exasol/errorcodecrawlermavenplugin/ExasolErrorCrawlerIT.java

```diff
@@ -0,0 +1,92 @@
+package com.exasol.errorcodecrawlermavenplugin;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.*;
+import java.util.List;
+import java.util.Objects;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.apache.maven.it.util.ResourceExtractor;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.io.TempDir;
+
+class ExasolErrorCrawlerIT {
```

Comment:
It would be nice a small explanation of what this test actually does, like:

"hey, we create a new brand maven project, we add some Classes with error message invocations and check those invocations are correctly analyzed" or similar.